ads-txt.php

```diff
@@ -22,6 +22,49 @@
 define( 'ADS_TXT_MANAGER_POST_OPTION', 'adstxt_post' );
 define( 'APP_ADS_TXT_MANAGER_POST_OPTION', 'app_adstxt_post' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @return string Minimum version required.
+ */
+function adstxt_minimum_php_requirement() {
+	return '7.4';
+}
+
+/**
+ * Whether PHP installation meets the minimum requirements
+ *
+ * @return bool True if meets minimum requirements, false otherwise.
+ */
+function adstxt_site_meets_php_requirements() {
+	return version_compare( phpversion(), adstxt_minimum_php_requirement(), '>=' );
+}
+
+// Ensuring our PHP version requirement is met first before loading plugin.
+if ( ! adstxt_site_meets_php_requirements() ) {
+	add_action(
+		'admin_notices',
+		function() {
+			?>
+			<div class="notice notice-error">
+				<p>
+					<?php
+					echo wp_kses_post(
+						sprintf(
+							/* translators: %s: Minimum required PHP version */
+							__( 'Ads.txt requires PHP version %s or later. Please upgrade PHP or disable the plugin.', 'ads-txt' ),
+							esc_html( adstxt_minimum_php_requirement() )
+						)
+					);
+					?>
+				</p>
+			</div>
+			<?php
+		}
+	);
+	return;
+}
+
 require_once __DIR__ . '/inc/post-type.php';
 require_once __DIR__ . '/inc/admin.php';
 require_once __DIR__ . '/inc/save.php';
```

Comment:
To avoid an issue similar to https://github.com/10up/simple-page-ordering/issues/163, the lines below this will need to be moved to a new file and moved to a file within the `inc` folder.

This will allow for the code to be updated to use modern PHP syntax without throwing a fatal error in older versions of PHP due to a syntax error.

Reply:
Code moved here:
- 81173e78940424157c700463e76efa6b61b6febb

Updates to function namespaces within new file:
- a38d118c320ebe201db973009bcb2921dd76802a

Unit test update:
- fda169948d04775da5ada293c4df0ef2e64a9fdb